File: python/problem-15(LastLevels)/LASTLEVELS.py
# cook your dish here
t = int(input())
for i in range(t):
    (x,y,z) = map(int,input().split())
    if x<=3:
        time_req = x*y
        print(time_req)
    else :
        if x%3==0:
            a=int((x/3)-1)
            b=a*z
            c=x*y
            print(b+c)
        else :
            a=int(x/3)
            b=a*z
            c=x*y
            print(b+c)

File: c++/problem-14(LastLevels)/LASTLEVELS.cpp
#include <iostream>
using namespace std;

int main() {
	// your code goes here
	int t;
	cin>>t;
	while (t--){
	    int x,y,z;
	    cin>>x>>y>>z;
	    if (x<=3){
	        int time_req=x*y;
	        cout<<time_req<<endl;
	    }
	    else{
	        if (x%3==0){
	            int a=(x/3)-1;
	            int b=x*y;
	            int c=a*z;
	            cout<<b+c<<endl;
	        }
	        else{
	            int a=x/3;
	            int b=x*y;
	            int c=a*z;
	            cout<<b+c<<endl;
	        }
	    }
	}
	
	return 0;
}
